Sources/CasePathsMacros/CasePathableMacro.swift

```diff
@@ -143,12 +143,13 @@ extension CasePathableMacro: MemberMacro {
       let indent =
         leadingTriviaLines
         .compactMap { $0.isEmpty ? nil : $0.prefix(while: \.isWhitespace).count }
-        .min(by: { (lhs: Int, rhs: Int) -> Bool in lhs == 0 ? lhs > rhs : lhs < rhs })
+        .min(by: { (lhs: Int, rhs: Int) -> Bool in lhs < rhs })
         ?? 0
       let leadingTrivia =
         leadingTriviaLines
         .map { String($0.dropFirst(indent)) }
         .joined(separator: "\n")
+        .trimmingSuffix(while: { $0.isWhitespace && !$0.isNewline })
```

Comment:
This ensures trailing whitespaces are trimmed from the trivia. It ensures no whitespaces are added after the final newline, potentially messing up the indentation of the property declaration.